contents/en/documents/install.html

```diff
@@ -52,7 +52,7 @@ <h3>Arch</h3>
     <h3>Alpine</h3>
     <pre><code>sudo apk add alsa-lib-dev libx11-dev libxrandr-dev libxcursor-dev libxinerama-dev libxi-dev mesa-dev pkgconf</code></pre>
     <h3>Void</h3>
-    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config</code></pre>
+    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config libXinerama-devel</code></pre>
```

Comment:
Please sort this like

```
sudo xbps-install libXcursor-devel libXi-devel libXinerama-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config
```

also please update content/ja/documents/install.html